src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -107,5 +107,5 @@
       "Size": 1904
     }
   },
-  "PackageSize": 2730619
+  "PackageSize": 2751099
```

Comment:
Seems odd & unexpected that "C++ modernization" would increate `.apk` size by ~20KB.

Reply:
There's more inlining and more standard library use (header-only classes), which also increases the amount of inlining and generated code.